app/assets/stylesheets/api-docs.scss

```diff
@@ -0,0 +1,18 @@
+code {
+  padding: 2px 4px;
+  font-family: monospace;
+  background-color: govuk-colour("light-grey");
+  color: govuk-colour("red");
+}
+
+.json-code-sample {
+  padding: 3px 5px;
+  font-family: Monaco, Consolas, "Lucida Console", monospace;
+  font-size: 15px;
+  color: $govuk-text-colour;
+  word-break: break-word;
+  word-wrap: break-word;
+  overflow-wrap: break-word;
+  white-space: break-spaces;
+  background: govuk-colour("light-grey");
+}
```

Comment:
Line ending please

Reply:
Fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/4133/commits/f2fb052d35e4f7c2da03676ba4b39a7fa9b3c3d7